### ISSUE TITLE
New fields not available in Frontend

### ISSUE THREAD

**Issue by robertpie** [derhansen/sf_event_mgt_extend_demo#2]
Hi Torben,

I am luckily using your extension sf_event_mgt. (running TYPO3 7.6.2)
I tried adding some fields with your demo extension. 
This works perfectly in the backend, but the new fields (e.g. "entrance") are not available in the frontend.

Could you give me a clue?

Thanks a lot,
Robert Pietschmann

**Comment by derhansen** [comment]
Hi Robert,

did you add the newly added fields to the template of sf_event_mgt? New fields must be added manually.

Best regards
Torben

**Comment by robertpie** [comment]
Hi Torben,

thanks for the replay.

I put a <f:debug>{events}/f:debug in the Lists.html – Template.

This outputs only the standard-fields:

```
  title => 'Workshop 1' (10 chars)

  teaser => 'sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam er
     at, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea reb
     um.' (155 chars)

  description => 'sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam er
     at, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea reb
     um. Stet clita k asd gubergren, no sea takimata. At vero eos et accusam et j
     usto duo dolores et ea rebum. sed diam voluptua. At vero eos et accusam et j
     usto. At vero eos et accusam et justo duo dolores et ea rebum. At vero eos e
     t accusam et justo duo dolores et ea rebum. sed diam voluptua. At vero eos e
     t accusam et justo duo dolores et ea rebum. sed diam nonumy eirmod tempor i
     nvidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero 
     eos et accusam et justo duo dolores et ea rebum. Stet clita k asd gubergren,
      no sea takimata. At vero eos et accusam et justo duo dolores et ea rebum. s
     ed diam voluptua. At vero eos et accusam et justo. At vero eos et accusam et
      justo duo dolores et ea rebum. At vero eos et accusam et justo duo dolores 
     et ea rebum. sed diam voluptua. At vero eos et accusam et justo duo dolores 
     et ea rebum. ' (1004 chars)

  program => 'Workshop' (8 chars)

  startdate => DateTimeprototypeobject (2016-03-16T09:44:00+01:00, 1458117840)

  enddate => DateTimeprototypeobject (2016-03-16T13:44:00+01:00, 1458132240)

  maxParticipants => 30 (integer)

  maxRegistrationsPerUser => 1 (integer)

  price => 120 (double)

  currency => 'EUR' (3 chars)

  category => TYPO3\CMS\Extbase\Persistence\ObjectStorageprototypeobject (2 items)

  registration => TYPO3\CMS\Extbase\Persistence\ObjectStorageprototypeobject (empty)

  registrationDeadline => DateTimeprototypeobject (2016-03-09T00:00:00+01:00, 1457478000)

  image => TYPO3\CMS\Extbase\Persistence\ObjectStorageprototypeobject (1 items)

  files => TYPO3\CMS\Extbase\Persistence\ObjectStorageprototypeobject (empty)

  youtube => '' (0 chars)

  location => DERHANSEN\SfEventMgt\Domain\Model\Locationprototypepersistent entity (uid=1, pid=107)

  enableRegistration => TRUE

  link => '' (0 chars)

  topEvent => FALSE

  additionalImage => TYPO3\CMS\Extbase\Persistence\ObjectStorageprototypeobject (empty)

  organisator =>  <javascript:;> DERHANSEN\SfEventMgt\Domain\Model\Organisatorprototypepersistent entitysee above (uid=1, pid=107)

  notifyAdmin => TRUE

  notifyOrganisator => TRUE

  enableCancel => FALSE

  cancelDeadline => NULL

  uniqueEmailCheck => FALSE

  uid => 1 (integer)

  _localizedUid => 1 (integer)modified

  _languageUid => -1 (integer)modified

  _versionedUid => 1 (integer)modified

  pid => 107 (integer)
```

Do I have to overwrite the event controller actions?

Best regards,

Robert 

Von: Torben Hansen [mailto:notifications@github.com] 
Gesendet: Sonntag, 14. Februar 2016 18:49
An: derhansen/sf_event_mgt_extend_demo
Cc: robertpie
Betreff: Re: [sf_event_mgt_extend_demo] New fields not available in Frontend (#2)

Hi Robert,

did you add the newly added fields to the template of sf_event_mgt? New fields must be added manually.

Best regards
Torben

—
Reply to this email directly or view it on GitHub https://github.com/derhansen/sf_event_mgt_extend_demo/issues/2#issuecomment-183937588 .  https://github.com/notifications/beacon/AQbvMidAcWopMLqos9ebL34tt_I1h-L9ks5pkLWQgaJpZM4HZ3cB.gif 

---

E-Mail ist virenfrei.
Von AVG überprüft - www.avg.de
Version: 2016.0.7442 / Virendatenbank: 4522/11608 - Ausgabedatum: 11.02.2016

**Comment by robertpie** [comment]
Hi Torben,

I have found a solution (Injecting the EventRepository).

Thanks for your help.

Best regards,

Robert

Von: Torben Hansen [mailto:notifications@github.com] 
Gesendet: Sonntag, 14. Februar 2016 18:49
An: derhansen/sf_event_mgt_extend_demo
Cc: robertpie
Betreff: Re: [sf_event_mgt_extend_demo] New fields not available in Frontend (#2)

Hi Robert,

did you add the newly added fields to the template of sf_event_mgt? New fields must be added manually.

Best regards
Torben

—
Reply to this email directly or view it on GitHub https://github.com/derhansen/sf_event_mgt_extend_demo/issues/2#issuecomment-183937588 .  https://github.com/notifications/beacon/AQbvMidAcWopMLqos9ebL34tt_I1h-L9ks5pkLWQgaJpZM4HZ3cB.gif 

---

E-Mail ist virenfrei.
Von AVG überprüft - www.avg.de
Version: 2016.0.7442 / Virendatenbank: 4522/11608 - Ausgabedatum: 11.02.2016
